src/features/myBeans/Page.js

```diff
@@ -44,13 +44,8 @@ const Page = () => {
 
   const { currentUser } = useAuth();
 
-  const showLoginMode = () => {
-    if (currentUser) {
-      return <BeanList />;
-    } else {
-      return <p> ログインされていないMyBeansページです</p>;
-    }
-  };
+  const getBeanList = () =>
+    currentUser ? <BeanList /> : <p> ログインされていないMyBeansページです</p>;
```

Comment:
```suggestion
  const getBeanList = () => {
    // TODO: ログインしていない場合に公開用のBeanListを返す
    currentUser ? <BeanList /> : <p> ログインされていないMyBeansページです</p>;
  }
```

今後手を入れる必要のある部分なので，TODO書いておいてください．